src/lib/discord/commando/commands/tracked.js

```diff
@@ -119,18 +119,18 @@ exports.run = async (client, msg, command) => {
 				let genderText = ''
 				let typeText = ''
 				if (!invasion.gender || invasion.gender === '') {
-					genderText = 'Gender: any, '
+					genderText = client.translator.translate('any')
```

Comment:
I think petap0w meant do `${client.translator.translate('Gender'):${client.translator.translate('any')`

Reply:
no it's ok the way he did it with gender in next line

Reply:
I think my solution is better than that. Tested, works